test/hooks/not-found.test.ts

```diff
@@ -1,36 +1,14 @@
 import chalk from '@oclif/color'
-import {expect, test} from '@oclif/test'
-import {cli} from 'cli-ux'
+import {test} from '@oclif/test'
 
 chalk.enabled = false
 
 describe('command_not_found', () => {
   test
-    .stub(cli, 'prompt', () => async () => 'y')
-    .stub(process, 'argv', [])
-    .stdout()
     .stderr()
     .hook('command_not_found', {id: 'commans'})
-    .catch('EEXIT: 0')
-    .end('runs hook with suggested command on yes', (ctx: any) => {
-      expect(ctx.stderr).to.be.contain('Warning: commans is not a @oclif/plugin-not-found command.\n')
-      expect(ctx.stdout).to.be.contain('commands\nhelp')
-    })
-
-  test
-    .stderr()
-    .stub(cli, 'prompt', () => async () => 'n')
-    .hook('command_not_found', {id: 'commans'})
-    .catch('Run @oclif/plugin-not-found help for a list of available commands.')
-    .end('runs hook with not found error on no', (ctx: any) => {
-      expect(ctx.stderr).to.be.contain('Warning: commans is not a @oclif/plugin-not-found command.\n')
-    })
-
-  test
-    .stderr()
-    .hook('command_not_found', {id: 'commans'})
-    .catch('Run @oclif/plugin-not-found help for a list of available commands.')
-    .end('runs hook with not found error after no input timeout', (ctx: any) => {
-      expect(ctx.stderr).to.be.contain('Warning: commans is not a @oclif/plugin-not-found command.\n')
-    })
+    .catch('commans is not a @binaris/oclif-plugin-not-found command.\n' +
```

Comment:
```suggestion
    .catch('command is not a @binaris/oclif-plugin-not-found command.\n' +
```
Probably also want to fix `id` on the line above

Reply:
This confused me at first as well. The typo of `commands` -> `commans` is intentional, would a different example `run` -> `rn` be better?

Reply:
Ouch, no, it just _is_ confusing.  Either add a comment or just leave it alone.